src/Command/RoutesCommand.php

```diff
@@ -74,6 +80,32 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
         $io->helper('table')->output($output);
         $io->out();
 
+        $duplicateRoutes = [];
+
+        // Check duplicate routes
+        foreach (Router::routes() as $route) {
```

Comment:
Instead of calling `Router::routes()` again save the return values to a variable for the first class above.